internal/service/networkpeering/resource_network_peering_test.go

```diff
@@ -148,12 +104,11 @@ func TestAccNetworkRSNetworkPeering_basicGCP(t *testing.T) {
 }
 
 func TestAccNetworkRSNetworkPeering_AWSDifferentRegionName(t *testing.T) {
-	acc.SkipTestForCI(t)
```

Comment:
Enabling this test since we can use `ProjectIDExecution`

Reply:
one question, would `ProjectIDExecution` be a safe usage here? If a new tests uses `ProjectIDExecution` they will share the same project, and I am not sure how this resource would behave in that case. I am seeing that for other tests in this resource we create a unique project for each one. (@lantoli might have additional inputs as he was looking into reusing projects in all our resources)

Reply:
True, I should have warned about that.
Basically, this means that if another test in this package uses the `ProjectIDExecution`, one of them will fail.
This is because the tests creates a `mongodbatlas_network_container` which can only be created **once** per project

Reply:
In this case I would favor the pattern used in other tests of defining the project in the terraform config. This way we ensure there is no risk of new test breaking existing ones. Usage of `ProjectIDExecution` will be limited to resources where a project can be safely shared for multiple tests executions. I can follow up with a PR to clarify this in our contributing guide.

Reply:
Updated in [bb0f628](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2094/commits/bb0f628f1a6d69b8b9e52b46eb234b4304edf68e) ✅